src/modules/update.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
+/**
+ * eslint-disable import/no-cycle
+ *
+ * @format
+ */
+
+/**
+ * eslint-disable import/no-cycle
+ *
+ * @format
+ */
+
+/**
+ * eslint-disable import/no-cycle
+ *
+ * @format
+ */
+
+/**
+ * eslint-disable import/no-cycle
+ *
+ * @format
+ */
+
+/**
+ * eslint-disable import/no-cycle
+ *
+ * @format
+ */
+
+/**
+ * eslint-disable import/no-cycle
+ *
+ * @format
+ */
+
+/** @format */
```

Comment:
- [ ] Please do not disable linter checks. The linter checks are here to help you improve the quality of your code.